### DISCUSSION TITLE
Welcome to J.A.R.V.I.S Discussions!

### DISCUSSION THREAD

**Discussion by GauravSingh9356** [GauravSingh9356/J.A.R.V.I.S#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mahirmsb25** [comment]
Hi man I have created jarvis and I have used open ai api as a brain and also added some tasks hu the thing is now I want my jarvis to create his tasks itself like when I order him to do a task and I have not added the code then jarvis will add it from somewhere and will do the task is it possible

**Comment by 1Floki0525** [comment]
is there a free API i can use for jarvis?
